templates/android/library/src/main/java/io/appwrite/Permissions.kt.twig

```diff
@@ -0,0 +1,21 @@
+package {{ sdk.namespace | caseDot }}
+
+class Permissions {
```

Comment:
I think this class should be called `Permission` rather than `Permissions` since each call to `Permissions.(read|write|create|update)` etc. creates a single permission and not permissions. Hence it makes more sense to use the singular form of the word

Reply:
Resolved.